docs/01_setup.md

```diff
@@ -145,13 +158,16 @@ cd apps/subgraph
 }
 ```
 
-4. Run `pnpm run build`. You can use env variables `NETWORK` and `VERSION` to switch config files.
-   - Create an `.env` file, and run the follow command in the console `export $(xargs < .env)`
-5. Run `graph auth --studio {key}`. You can find the key in subgraph studio dashboard.
-6. Run `pnpm run deploy` to deploy subgraph
+4. If you haven't installed the graph CLI in your computer globally yet, do so by running `pnpm i -g @graphprotocol/graph-cli`
+5. Run `pnpm run build`. You can use env variables `NETWORK` and `VERSION` to switch config files.
+   - Create an `.env` file, and run the following command in the console `export $(xargs < .env)`
+6. Copy the authenticate command from the 'Auth and Deploy' section in the Subgraph Studio graph details, paste it into the terminal, and execute it. The command should look like this: `graph auth --studio {key}`.
+7. Change `package.json` deploy script to `graph deploy --node https://api.studio.thegraph.com/deploy/ <name>` where <name> is the name that you gave to the subgraph you created.
+8. Run `pnpm run deploy` to deploy the subgraph.
+9. Note down the url of the deployed subgraph API endpoint.
 
 > [!IMPORTANT]
-> The `pnpm run deploy` command call `maci-subgraph` as the subgraph name as default, but if you named your subgraph differently (e.g. maci-graph, my-graph, etc.), please change the command to `graph deploy --node https://api.studio.thegraph.com/deploy/ :your_subgraph_name`
+> If you always use the latest version the URL will be `https://api.studio.thegraph.com/query/[some number]/[name]/version/latest`.
```

Comment:
I checked out what does the number mean, it's the `USER_ID`, so this URL should be
`https://api.studio.thegraph.com/query/<USER_ID>/<YOUR_SUBGRAPH_NAME>/version/latest`